.ci/config/config.ssl.json

```diff
@@ -4,7 +4,11 @@
       "PasswordlessUser": "no_password",
       "SecondaryDatabase": "testdb2",
       "SupportedFeatures": "Json,StoredProcedures,Sha256Password,LargePackets",
+	  
+      "TestData": "../../../../TestData",
       "MySqlBulkLoaderLocalCsvFile": "%TESTDATA%/LoadData_UTF8_BOM_Unix.CSV",
-      "MySqlBulkLoaderLocalTsvFile": "%TESTDATA%/LoadData_UTF8_BOM_Unix.TSV"
+      "MySqlBulkLoaderLocalTsvFile": "%TESTDATA%/LoadData_UTF8_BOM_Unix.TSV",
+	  
+	  "CertificatesPath": "..\\..\\..\\..\\..\\.ci\\server\\certs"
```

Comment:
let's stick to `../` for config file, mixing `..\` is confusing 

also nitpick- could you use the same size indent